src/encoder.rs

```diff
@@ -1220,7 +1220,11 @@ pub fn encode_tx_block<T: Pixel>(
     // Store tx-domain distortion of this block
     tx_dist = coeffs
       .iter()
-      .zip(rcoeffs)
+      .zip(
```

Comment:
Thanks for fixing this! The 64x64 transformed original source in coeffs is kept w/o zeroing three 32x32 quadrants, but the reconstructed 64x64 rcoeffs (i.e. after dequant + inverse-tx) should have all zeros in those three 32x32 quadrants.